user-guide/python.rst

```diff
@@ -1,281 +1,106 @@
 Using Python
 ============
 
-Python on Cirrus is provided by the `Anaconda <https://www.continuum.io/>`__
-distribution. Both Python 2 and Python 3 versions of the distributions are
-supported.
+Python for general users
+------------------------
 
-The central installation provides many of the most common packges used for
-scientific computation and data analysis.
+Cray Python 3 distribution
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If the packages you require are not included in the central Anaconda Python
-distribution, then the simplest way to make these available is often to install
-your own version of `Miniconda <https://conda.io/miniconda.html>`__  and add the packages you need. We provide 
-instructions on how to do this below. An alternative way to provide your own
-packages (and to make them available more generally to other people in your
-project and beyond) would be to use a Singularity container, see the :doc:`containers`
-chapter of this User Guide for more information on this topic.
+Python on ARCHER2 is provided by Cray. Both Python 2 and Python 3 versions of
```

Comment:
Only Python 3 will be provided by Cray so we should note this. Python 2 is beyond EOL and we will likely not provide any useful environment for it on ARCHER2.

Remove the comment on Cray providing Python 2

Reply:
I've removed the reference to Python 2